arcgis-ios-sdk-samples/Display information/Create symbol styles from web styles/CreateSymbolStylesFromWebStylesViewController.swift

```diff
@@ -0,0 +1,273 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class CreateSymbolStylesFromWebStylesViewController: UIViewController, UIAdaptivePresentationControllerDelegate {
+    // MARK: Storyboard views
+    
+    /// The map view managed by the view controller.
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = makeMap(referenceScale: 1e5)
+            mapView.setViewpoint(
+                AGSViewpoint(
+                    latitude: 34.28301,
+                    longitude: -118.44186,
+                    scale: 1e4
+                )
+            )
+        }
+    }
+    /// The button to show legend table.
+    @IBOutlet var legendBarButtonItem: UIBarButtonItem!
+    
+    // MARK: Instance properties
+    
+    /// The Esri 2D point symbol style created from a web style.
+    let symbolStyle = AGSSymbolStyle(styleName: "Esri2DPointSymbolsStyle", portal: .arcGISOnline(withLoginRequired: false))
```

Comment:
This doesn't need to be a property either. I can just be created in `getSymbols` since it is only used there, right?

Reply:
I'm not really sure why, but the `AGSSymbolStyle.symbol(forKeys:completion:)` method doesn't callback if I don't hold a strong reference to the `symbolStyle`, e.g. if I create a local variable in the `viewDidLoad` method. 🤔

Reply:
Oh, duh. I knew that. Yes, the property is needed to prevent the object from being deallocated while fetching the symbols. Never mind.